tests/test_holoclean_repair.py

```diff
@@ -0,0 +1,37 @@
+import unittest
```

Comment:
Hmmm so I disagree that we should keep these as scripts. If we wanted to have example scripts we should have them in a different folder not called `tests` (e.g. in `tutorials` or `examples`).

Making these `unittest.TestCase`s will let us run integration CI tests or if we build these as `conda` packages. 

Because of 
```
if __name__ == '__main__':
    unitttest.main()
```
these scripts functionally behave as "naked" scripts anyways. I copy this over into another file but I don't think we should remove these tests because it adds a few extra boilerplate lines.

Reply:
Change to `examples` then, and leave the `test_holoclean_fusion.py` script in `tests` since it actually tests something, but running new datasets or changing parameters is not a unit test.

Reply:
Done 👍 also moved test data to `testdata/`